src/client/components/filterToggler/filterToggler.test.js

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import { FilterToggler } from './filterToggler';
+
+describe('Filter Toggler', () => {
+  const props = {
+    audioData: {
+      sound: {
+        effects: ['Delay', 'Flanger', 'Reverb', 'Tremolo', 'Stereo panner'],
+        removeEffect: jest.fn(),
+        addEffect: jest.fn(),
+      },
+      voice: {
+        effects: [],
+        removeEffect: jest.fn(),
+        addEffect: jest.fn(),
+      },
+    },
+    blocksData: [
+      {
+        name: 'Delay',
+        isVisible: true,
+      },
+      {
+        name: 'Ping-Pong Delay',
+        isVisible: false,
+      },
+      {
+        name: 'Dub Delay',
+        isVisible: false,
+      },
+      {
+        name: 'Distortion',
+        isVisible: false,
+      },
+      {
+        name: 'Quadrafuzz',
+        isVisible: false,
+      },
+      {
+        name: 'Flanger',
+        isVisible: true,
+      },
+      {
+        name: 'Reverb',
+        isVisible: true,
+      },
+      {
+        name: 'Tremolo',
+        isVisible: true,
+      },
+      {
+        name: 'Stereo panner',
+        isVisible: true,
+      },
+      {
+        name: 'Compressor',
+        isVisible: false,
+      },
+      {
+        name: 'Low-Pass Filter',
+        isVisible: false,
+      },
+      {
+        name: 'High-Pass Filter',
+        isVisible: false,
+      },
+      {
+        name: 'Ring Modulator',
+        isVisible: false,
+      },
+    ],
+  };
+
+  it('should render filterToggler element', () => {
+    const nextProps = {
+      ...props,
+    };
```

Comment:
Why do you need here this "nextProps" constant? You can use just your "props" constant, can't you?